src/Sleep.js

```diff
@@ -17,7 +17,13 @@ class Sleep {
 
   calculateDailySleep(id, date) {
     const findSleepByDate = this.sleepData.find((data) => id === data.userID && date === data.date);
-    return findSleepByDate.hoursSlept;
+    // this.sleepData.forEach(data => {
+    //   typeof data.hoursSlept === "number" ? data.hoursSlept : 0
```

Comment:
I think we were trying to deal with bad data. I started a new branch that added conditionals to deal with bad data in sleep and activity class.